package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ng-adventure",
+  "version": "1.0.0",
+  "description": "ng adpatation of Zork like text based game",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Drew",
+  "license": "ISC",
+  "devDependencies": {
+    "webpack-stream": "^3.2.0"
```

Comment:
Don't forget to --save-dev or -D gulp when you install it.  That way it will be in your package as a devDependency.